features/kube_features.go

```diff
@@ -0,0 +1,43 @@
+/*
```

Comment:
nit: s/kube_features/features/ in the file name. The name `kube_features` makes me think that it's related to kubernetes features rather than feature gate definitions for cluster api.